canary/src/main/kotlin/sw2025/canary/global/security/jwt/JwtProvider.kt

```
@@ -0,0 +1,108 @@
+        val userDetails: UserDetails = getDetails(getJws(token))
+        return UsernamePasswordAuthenticationToken(userDetails, "", userDetails.authorities)
+    }
+
+    private fun getJws(token: String): Claims {
+        return try {
+            Jwts.parser()
+                .verifyWith(secretKey)
+                .build()
+                .parseSignedClaims(token)
+                .payload
+        } catch (e: ExpiredTokenException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)